digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/process/DefaultSectionCalculatorStrategySpec.scala

```diff
@@ -282,6 +279,36 @@ class DefaultSectionCalculatorStrategySpec extends FunSuite with Matchers {
     }
   }
 
+  test("Test findStartingPoints " +
+       "When a combined road has a loopend" +
+       "Then startingpoint from triple connection should be found.") {
+    runWithRollback {
+      val geom1 = Seq(Point(372017, 6669721), Point(371826, 6669765))
```

Comment:
kommentoitko tähän näiden ylle, tai ensimmäisen rivin loppuun:
// Point(371826, 6669765) found in three geometries

(Tai jos haluaa vielä selkeämmäksi, niin ton kyseisen Pointin vois napsia omaan muuttujaansa, ja sit vaan sijoitella noille riveille)

Reply:
Teit molemmat :D <3